pandas/sparse/series.py

```diff
@@ -654,6 +656,47 @@ def combine_first(self, other):
         dense_combined = self.to_dense().combine_first(other)
         return dense_combined.to_sparse(fill_value=self.fill_value)
 
+    def to_coo(self, ilevels=(0,), jlevels=(1,), sort_labels=False):
+        """
+        Create a scipy.sparse.coo_matrix from a SparseSeries with MultiIndex.
+        Use ilevels and jlevels to determine the row and column coordinates respectively.
+        ilevels and jlevels are the names (labels) or numbers of the levels.
+        {ilevels, jlevels} must be a partition of the MultiIndex level names (or numbers).
+
+        Parameters
+        ----------
+        ilevels: tuple/list
+        jlevels: tuple/list
+        sort_labels: bool
+
+        Returns
+        -------
+        y : scipy.sparse.coo_matrix
+        il : list (row labels)
+        jl : list (column labels)
+        """
+        A, il, jl = sparse_series_to_coo(self, ilevels, jlevels, sort_labels=sort_labels)
+        return(A, il, jl)
+
+    @classmethod
+    def from_coo(cls, A, dense_index=False):
+        """
+        Create a SparseSeries from a scipy.sparse.coo_matrix.
+        If dense_index is False (default), the SparseSeries will only
+        contain the non-null entries of the original coo_matrix (the SparseSeries will
+        in fact be dense in the sense that no entriess will be masked).
+        If dense_index is True, the index of the SparseSeries
+        consists of the sorted (row, col) coordinates of the coo_matrix.
+
+        Parameters
+        ----------
+        A: scipy.sparse.coo_matrix
+
+        Returns
+        -------
+        s: SparseSeries
+        """
+        return(coo_to_sparse_series(A, dense_index=dense_index))
 # overwrite series methods with unaccelerated versions
```

Comment:
I meant between line 699 and 700 above, so there is a blank line between the end of the function definition and the following thing (in fact, following PEP8, there could even be two blank lines, as it is the end of the class definition here, but that is a bit nitpicking)